features/car/give_way_sign_penalties.feature

```diff
@@ -0,0 +1,77 @@
+@routing @car @give_way_sign
+Feature: Car - Handle give way signs
+
+    Background:
+        Given the profile "car"
+
+    Scenario: Car - Encounters a give way sign
```

Comment:
It is almost copy-paste of existing tests for traffic lights https://github.com/Project-OSRM/osrm-backend/blob/afe34456b9993d465d37ea3898fee1db6a2b5561/features/car/traffic_light_penalties.feature#L7
Probably worth having here all tests we have for traffic lights, but is there way to reduce copy-paste here somehow? E.g. somehow run the same test, but with different `highway` tag values for each run?

Reply:
I expect there's some support for parametrization of tests, but we're on an ancient Cucumber version, so might take a lot of effort. Plus, having a separate file per penalty feature will allow the functionality to diverge if required.